src/shared/firewall-util-nft.c

```diff
@@ -1049,6 +1049,8 @@ int fw_nftables_add_local_dnat(
                 int tmp = fw_nftables_recreate_table(ctx->nfnl, af, transaction, tsize);
 
                 if (tmp == 0) {
+                        /* table created anew; previous address already gone */
```

Comment:
`anew` -> `a new`?

Reply:
anew is a correct english word. this looks right the way it is actually.

Reply:
Oh, I didn't know that word...Thanks.